src/plugins/plugin.tooltip.js

```diff
@@ -630,13 +630,13 @@ export class Tooltip extends Element {
     return tooltipItems;
   }
 
-  update(changed, replay) {
+  update(changed, replay, inChartArea = true) {
     const options = this.options.setContext(this.getContext());
     const active = this._active;
     let properties;
     let tooltipItems = [];
 
-    if (!active.length) {
+    if (!active.length || !inChartArea) {
```

Comment:
It seems that this behaviour was added intentionally and is not a bug when I look at the code and the [pr](https://github.com/chartjs/Chart.js/pull/9970) where this got introduced.

I will look at this later to see if this might break any implementations that rely on that behaviour

![image](https://github.com/user-attachments/assets/ef94a591-b9c9-4fc7-9b5a-9e4ed724a41b)

Reply:
Thanks for the feedback.
Correct me if I'm wrong, from my understanding, the `!active.length` is added to make sure the chart can be interactive even when the cursor is out of it (hovering over the legend and high light the corresponding data in the chart). 
Adding the extra check will only effect the visibility of the tooltip in this scenario.

Reply:
Hi @LeeLenaleee, do you think this is a proper change or do you have any further comments on this?

Reply:
Hi, I tested the changes, what you said is correct, this does still work with externally calling code to `setActiveElements`. Although the name is kind of misleading in those cases. Since the bool is called `inChartArea`, but when you use `setActiveElements` you are not in the chart area.

Also this change seems to introduce a new bug. When you move your mouse to the legend quickly and return to the same slice the hover does not trigger anymore

Reply:
Thanks @LeeLenaleee , I tested it and tried to fix that, but looks like it's not a simple change. I will change this PR to draft and work on this continuously.